tx.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+/*
+message Tx {
+    oneof tx {
+        WitnessTx witness = 1;
+        SignTx    sign    = 2;
+        RelayTx   relay   = 3;
+    }
+}
+
+message RelayTx {
+    
+}
+
+message SignTx {
+
+}
+
+*/
+
+message WitnessTx {
+    bytes signature = 1;
+    uint64 sequence = 2;
+    oneof tx {
+        LockMsg lock = 3;
+    }
+}
+
+message LockMsg {
```

Comment:
What's the difference between a LockMsg and a WitnessTx?

Reply:
WitnessTx was intended to be a wrapper for LockMsg and BurnMsg. Since we are working with LockMsg for now, WitnessTx will only contain LockMsg.

Also, WitnessTx corresponds to sdk.Tx, LockMsg corresponds to sdk.Msg.